build.gradle

```diff
@@ -37,7 +37,7 @@ googleJavaFormat {
 }
 
 group = 'com.uber.cadence'
-version = '2.3.0'
+version = '2.3.1'
```

Comment:
Would you update the release notes as well including my previous change as well?

Reply:
Yes. Will update for both.